src/app/cli/src/commands/inspect_schema_command.rs

```diff
@@ -97,44 +97,51 @@ impl InspectSchemaCommand {
         Ok(())
     }
 
-    fn print_schema_json(&self, schema: &Type) -> Result<(), CLIError> {
-        kamu_data_utils::schema::format::write_schema_parquet_json(&mut std::io::stdout(), schema)?;
-        Ok(())
+    fn query_errors(e: QueryError) -> CLIError {
+        match e {
+            QueryError::DatasetNotFound(e) => CLIError::usage_error_from(e),
+            QueryError::DatasetSchemaNotAvailable(_) => unreachable!(),
+            e @ (QueryError::DataFusionError(_) | QueryError::Access(_)) => CLIError::failure(e),
+            e @ QueryError::Internal(_) => CLIError::critical(e),
+        }
     }
 
-    fn print_schema_unavailable(&self) {
-        eprintln!(
-            "{}: Dataset schema is not yet available: {}",
-            console::style("Warning").yellow(),
-            self.dataset_ref.alias().unwrap(),
-        );
+    async fn get_parquet_schema(&mut self) -> Result<Type, CLIError> {
+        Ok(self
+            .query_svc
+            .get_schema_parquet(&self.dataset_ref)
+            .await
+            .map_err(Self::query_errors)?
+            .unwrap())
     }
 }
 
-#[async_trait::async_trait(?Send)]
+#[async_trait::async_trait(? Send)]
 impl Command for InspectSchemaCommand {
     async fn run(&mut self) -> Result<(), CLIError> {
-        let maybe_schema =
-            self.query_svc
-                .get_schema(&self.dataset_ref)
-                .await
-                .map_err(|e| match e {
-                    QueryError::DatasetNotFound(e) => CLIError::usage_error_from(e),
-                    QueryError::DatasetSchemaNotAvailable(_) => unreachable!(),
-                    e @ (QueryError::DataFusionError(_) | QueryError::Access(_)) => {
-                        CLIError::failure(e)
-                    }
-                    e @ QueryError::Internal(_) => CLIError::critical(e),
-                })?;
-
-        match maybe_schema {
-            Some(schema) => match self.output_format.as_deref() {
-                None | Some("ddl") => self.print_schema_ddl(&schema),
-                Some("parquet") => self.print_schema_parquet(&schema)?,
-                Some("json") => self.print_schema_json(&schema)?,
-                _ => unreachable!(),
-            },
-            None => self.print_schema_unavailable(),
+        match self.output_format.as_deref() {
+            None | Some("ddl") => {
+                let schema = self.get_parquet_schema().await?;
+                self.print_schema_ddl(&schema);
+            }
+            Some("parquet-json") => {
```

Comment:
Why did we lose the `"parquet"` format?

Reply:
Actually, if I read this correctly, if you do `kamu inspect schema dataset --output-format parquet-json` looks like you will get Parquet's custom format...

Please re-test this command with all combinations of `output-format`

Reply:
from the task...Update kamu instpect schema command
rename json format to parquet-json
introduce arrow-json output format

Reply:
parquet: <img width="1136" alt="image" src="https://github.com/kamu-data/kamu-cli/assets/3535553/1111ca45-8551-4d53-b627-9464c8aeff6f">
parquet-json: <img width="1659" alt="image" src="https://github.com/kamu-data/kamu-cli/assets/3535553/df6ecac4-be64-4d03-ad40-0cd34b97581c">
ddl:<img width="1129" alt="image" src="https://github.com/kamu-data/kamu-cli/assets/3535553/4fbfe2a7-0df3-498b-8ba3-aee265ffae8e">
arrow-json:
<img width="1644" alt="image" src="https://github.com/kamu-data/kamu-cli/assets/3535553/a8ede7cf-df6b-4331-b61e-0b1f6d4970d3">

Reply:
done